src/Symfony/Bundle/WebProfilerBundle/Twig/WebProfilerExtension.php

```diff
@@ -72,6 +76,7 @@ public function getFunctions()
         return array(
             new \Twig_SimpleFunction('profiler_dump', $profilerDump, array('is_safe' => array('html'), 'needs_environment' => true)),
             new \Twig_SimpleFunction('profiler_dump_log', array($this, 'dumpLog'), array('is_safe' => array('html'), 'needs_environment' => true)),
+            new \Twig_SimpleFunction('help', array($this, 'help'), array('is_safe' => array('html'))),
```

Comment:
I would propose to call this `help_attrs` or the like, to cover the usage of the method more (calling it attrs makes it very clear that this has to be executed inside a tag).